app/controllers/state_file/questions/nj_homeowner_eligibility_controller.rb

```diff
@@ -6,20 +6,25 @@ class NjHomeownerEligibilityController < QuestionsController
       before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
 
       def self.show?(intake)
-        intake.household_rent_own_own?
+        intake.household_rent_own_own? &&
+        Efile::Nj::NjPropertyTaxEligibility.determine_eligibility(intake) != Efile::Nj::NjPropertyTaxEligibility::INELIGIBLE
       end
 
       def next_path
         options = {}
         options[:return_to_review] = params[:return_to_review] if params[:return_to_review].present?
-
+        
         case StateFile::NjHomeownerEligibilityHelper.determine_eligibility(current_intake)
         when StateFile::NjHomeownerEligibilityHelper::INELIGIBLE
           NjIneligiblePropertyTaxController.to_path_helper(options)
         when StateFile::NjHomeownerEligibilityHelper::UNSUPPORTED
           NjUnsupportedPropertyTaxController.to_path_helper(options)
         else
-          NjHomeownerPropertyTaxController.to_path_helper(options)
+          if Efile::Nj::NjPropertyTaxEligibility.determine_eligibility(current_intake) == Efile::Nj::NjPropertyTaxEligibility::POSSIBLY_ELIGIBLE_FOR_CREDIT
```

Comment:
would it be cleaner/easier to read if there a way to just call `current_intake.possibly_eligible_for_credit?` instead of doing comparisons like this everywhere? i.e. define these methods on NJ intake?

Reply:
Ooh thanks for this suggestion. I don't feel too strongly about having the logic live in the util vs intake model, especially since this is my first PR so there may be a pattern I'm not aware of, but I lean util to keep it separate from everything else.

That being said I'll make the functions like `.possibly_eligible_for_credit?(intake)` in the util, so it can be

```
Efile::Nj::NjPropertyTaxEligibility.possibly_eligible_for_deduction_or_credit?(intake)
```

instead of

```
Efile::Nj::NjPropertyTaxEligibility.determine_eligibility(intake) == Efile::Nj::NjPropertyTaxEligibility::POSSIBLY_ELIGIBLE_FOR_DEDUCTION_OR_CREDIT
```

Reply:
sounds fine to use the util

Reply:
added these in [the latest commit](https://github.com/codeforamerica/vita-min/pull/4941/commits/65719ab69a0cfe2c459421dab6ad90f2dd9f96e3)

Reply:
Thanks for humoring me!